label_studio_ml/utils.py

```diff
@@ -39,8 +39,12 @@ def get_all_classes_inherited_LabelStudioMLBase(script_file):
         if name == LabelStudioMLBase.__name__:
             continue
         if issubclass(obj, LabelStudioMLBase):
-            names.append(name)
+            names.add(name)
+        for base in obj.__bases__:
+            if LabelStudioMLBase.__name__ == base.__name__:
```

Comment:
Could you please explain what makes difference is and how it fixes the issue?

Reply:
In some versions of python issubclass(obj, LabelStudioMLBase) doesn't return true for inherited class if it was imported before. 
Switching to set and collecting subclass with different methods allows to fix this issue.